src/Material/Textfield.elm

```diff
@@ -282,25 +280,29 @@ defaultModel =
 
 {-| Component actions. `Input` carries the new value of the field.
 -}
-type Msg
+type Msg m
   = Blur
   | Focus
   | Input String
+  | Multi (Dispatch.Msg m)
```

Comment:
This is also something that could possible be changed. Do we need to be specific to have `Dispatch.Msg m`  or could we just have `Dispatch` work on any message of the type `Dispatch (List Msg)`